Zend/zend_object_handlers.c

```diff
@@ -1184,12 +1184,22 @@ found:;
 	return variable_ptr;
 
 lazy_init:
```

Comment:
Nit: Usually, the restriction to declare a variable after a label is circumvented with a `;`.

```suggestion
lazy_init:;
```

If you prefer braces, that's fine too.